CodeChef_problems/AND Plus OR/solution.cpp

```diff
@@ -0,0 +1,24 @@
+#include<bits/stdc++.h>
```

Comment:
Please check your solution again.
For the input 
```
2
1
8
```
Your code outputs
```
0 1
0 8
```
Expected correct output:
```
0 1
5 3
```

Reply:
Sir, I have already mentioned in the comments that the problem belongs to constructive algorithm , and there are many possible solution of the single problem, you could even check my solution correctness by submitting it in Codechef .

- [ ]